server/service/integration_live_queries_test.go

```diff
@@ -961,8 +961,8 @@ func (s *liveQueriesTestSuite) TestCreateDistributedQueryCampaign() {
 	}
 	s.DoJSON("POST", "/api/latest/fleet/queries/run", req, http.StatusBadRequest, &createResp)
 
-	// wait a second to prevent duplicate name for new query
-	time.Sleep(time.Second)
+	// wait to prevent duplicate name for new query
+	time.Sleep(200 * time.Millisecond)
```

Comment:
I'm guessing you checked several times to make sure 200ms will not cause occasional failure.

Reply:
Yes. If we see a fail in CI, I would like to use a check loop instead of increasing sleep time.

Reply:
I like this attitude. we shouldn't add unnecessary time to U.Tests.
BTW, are we using t.parallel in our tests? worth looking at it to decrease run time

Reply:
I've been using parallel in my tests. But yes, there are probably places we can add it.